config/jenkins-jobs_infra.ci.jenkins.io.yaml

```diff
@@ -101,6 +101,9 @@ jobsDefinition:
         branchIncludes: "production staging updatecli_* PR-* main"
       shared-tools:
         name: Shared Tools
+      pipeline-library:
+        name: Shared Pipeline Library
+        jenkinsfilePath: Jenkinsfile
```

Comment:
```suggestion
```

Reply:
Unneeded: we only want updatecli

Reply:
I thought that was mandatory by process, if we manage the updatecli to also have a "standard" build ... but I agree, it's useless as already built on ci.jenkins.io

Reply:
removed